### PR TITLE
Vue frontend

### PR DESCRIPTION
**Pull request by Just4Ease**

### CODE REVIEW

File: src/components/Archives.vue [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-  <h1>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Iure fuga sequi eum esse harum quos quod, sit dolores suscipit eaque blanditiis ad cupiditate, nesciunt velit expedita libero inventore ducimus. Maiores?</h1>
+  <div id="content">
+
+  </div>
 </template>
 
 <script>
 export default {
-
+name: 'Archives'
```
Comment (humanityjs) [inline src/components/Archives.vue:9]: Add 2 space before `name`

File: src/components/Contribute.vue [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<template>
+  <div id="content"></div>
+</template>
+
+<script>
+export default {
+name: 'Contribute'
```
Comment (humanityjs) [inline src/components/Contribute.vue:7]: Add 2 spaces before `name`

File: src/components/Contribute.vue [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+}
+</script>
+
+<style>
```
Comment (humanityjs) [inline src/components/Contribute.vue:11]: You can remove the style tag since you are not making use of it

File: src/components/Index.vue [hunk 4]
```diff
@@ -1,169 +1,220 @@
...
 <script>
+  import countryStates from "@/utils/geolocation";
+
   export default {
-    name: 'Index'
+    name: "Index",
+    data() {
+      return {
+        countryStates,
+        latest_missing_people: [{
```
Comment (humanityjs) [inline src/components/Index.vue:135]: Is it possible to have this data in a json file and then import it here just to minimise space.

File: src/components/Index.vue [hunk 5]
```diff
@@ -1,169 +1,220 @@
...
+            age: '22yrs',
+          }
+        ]
+      };
+    }
   };
 
 </script>
 
 <style>
```
Comment (humanityjs) [inline src/components/Index.vue:217]: You can remove the style tag completely

File: src/components/ReportCase.vue [hunk 6]
```diff
@@ -1,10 +1,10 @@
 <template>
-  <h1>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cumque laudantium alias ea, sed reprehenderit vero eveniet iusto labore vel tenetur ipsam dicta id. Illo molestiae fuga, tempora facere officia praesentium.</h1>
+  <div id="content"></div>
 </template>
 
 <script>
 export default {
-
+name: 'ReportCase'
```
Comment (humanityjs) [inline src/components/ReportCase.vue:7]: Require 2 space (1 tab) before `name`

File: src/components/ReportSightings.vue [hunk 7]
```diff
@@ -1,35 +1,15 @@
...
   </div>
 </template>
 
 <script>
-  export default {
-    name: 'ReportSighting'
-  }
-
+export default {
+name: 'ReportSightings'
```
Comment (humanityjs) [inline src/components/ReportSightings.vue:9]: Same here

**Review by humanityjs — COMMENTED** [review]
Looks good. Great work 👍

**Comment by humanityjs** [comment]
Here are some things I see that can use a little bit of retouching. 

ot8g0y

Based on the image above, can we make the banner take full screen (100vh) and then make the highlighted part centered vertically? I think it will make more sense if we do that.
We can also add an overlay to the image to improve the text quality.

rlxijh

Based on the highlighted part above, we can add a background color to header to prevent text overlapping on scroll.
Also, the text block before the missing people content could be improved to be more catchy. I love good designs and most people do. And trust me, I'll always come back to a site because of the UI/UX even if I have no business with the site.

Overall, we can improve the font and the font-family used throughout the page/website. 

Nice job by the way. You are the real MVP!

**Comment by Just4Ease** [comment]
Thanks People.
Sorry I've not even pushed something new,
Had issues with internet connection,
They metered it. FUP shii.

**Pull request merged**